tests/test_pypi.py

```diff
@@ -423,7 +423,7 @@ def test_normalize_requirements_list():
     config = Configuration(name="pytest")
     requirements = ["pytest ~=5.3.2", "pyqt5"]
     requirements = set(normalize_requirements_list(requirements, config))
-    expected = {"pytest >=5.3.2,==5.3.*", "pyqt"}
+    expected = {"pytest ~=5.3.2", "pyqt"}
```

Comment:
cc @ocefpaf 
what do you think? should we revert to use ``~=`` as conda-build supports it https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#build-version-spec

Reply:
Actually the `~=` operator is not documented in that link, but it is supported. See https://github.com/conda/conda-build/issues/4553.

Reply:
I don't have a strong opinion. My take is, if conda-build supports it and it is a python standard we should too. In the past some folks complained that parts of the bot (I don't really remember) choked on this syntax. Maybe it is worth asking in the core gitter is someone remembers what was wrong with recipes that used that syntax.

Reply:
Thanks @ocefpaf for the feedback. Regarding potential problems, it is necessary to use `conda-build >=3.20.5`. We have already added this to Grayskull's `run_constrained`.

It would be great if you could ask in the core gitter to check if there are any further known issues with `~=`. Thanks so much for looking into this!

Reply:
cool, lets get it merged then! :tada: